### ISSUE TITLE
Barcode lines and background colors

### ISSUE THREAD

**Issue by LuigiQuarta** [Zoimaru/ng-barcode#2]
Add an option to configure the barcode lines and background colors

**Comment by lorenzodianni** [comment]
Config added for Code39: [Color preference for Code39](https://github.com/Zoimaru/ng-barcode/issues/8) :muscle:

**Comment by itmilos** [comment]
I am trying to add random color button that will randomly assign colors to every line of the bar code.

**Comment by lorenzodianni** [comment]
@itmilos you can't color each line, as you can see on demo page, you can change the color for all barcode's lines. That because a color for each line could compromise the correct barcode lecture by scanners

**Comment by itmilos** [comment]
Indeed, but if you can color every line after the barcode is generated what would do the trick?
